joomla/joomla-cms
# [4.2] Convert mod_articles_latest to new structure

Pull Request for Issue # .

### Summary of Changes
This PR converts mod_articles_latest  to new structure

### Testing Instructions
1. Use Joomla 4.2
2. Create an instance of Articles - Latest module. Note how it works.
3. Apply patch
4. Delete administrator/cache/autoload_psr4.php
5. Refresh the page, make sure the module still displays the same as before.
